charts/prometheus-federator/0.1.1/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+annotations:
```

Comment:
This is the new chart introduced that contains the changes in https://github.com/rancher/charts/pull/1991 as well as a fix to a typo that prevented installs in private registries (the issue linked in the PR description).